docs/source/dynamic-outputs.rst

```diff
@@ -0,0 +1,87 @@
+Dynamic Outputs
+===============
+
+Overview
+--------
+
+Adds the ability to have custom logic run to define an ``output`` based on information within the ``record``.
+For information on supported outputs and how to add support for additional outputs, see `outputs <outputs.html>`_
+
+As can be seen by the examples below, they are easy to configure, but add a very useful feature to StreamAlert. With the simple addition of a
+`lookup-table <lookup-tables.html>`_ you can take a rule like ``cloudtrail_root_account_usage`` and configure it as such:
```

Comment:
nitpick 
Using LookupTables alongside Dynamic Outputs is a great (and _intended_) way of using it. However, maybe include LookupTables as a part 2 in the documentation? Because you **have** to use Dynamic Outputs with LookupTables. You can use a simple switch statement too. 

This would be to reduce the (perceived) barrier to entry, maybe increase the chance for users to adopt Dynamic Outputs without thinking _"oh man, I have to configure LookupTables too?"_ Just my 2cents

Reply:
Will update and push tomorrow (running on UK time) but thanks for the feedback!